otherMethods.py

```diff
@@ -49,45 +49,55 @@ def download(cls, api, datasetName, entityIds, productName, output_dir):
         results_list = []
         for availableDownload in availableDownloads:
             url = availableDownload['url']
-            path = cls._download(url, output_dir)
+            if (productName.find('Bundle') == -1):

```

Comment:
The brackets are unnecessary:
 if productName.find('Bundle') == -1:

Reply:
My bad! It became a habit for me to do that :)